packages/app/src/templates/components/templates-list/templates-list.component.tsx

```diff
@@ -40,53 +48,108 @@ export const TemplatesList: FunctionComponent<TemplatesListProps> = ({
   deleteTemplates,
   updateTemplate,
 }) => {
+  const [templatesList, setTemplateList] = useState<Template[]>(templates)
+
+  const reorder = (
```

Comment:
Let's mark it as a temporary solution

Reply:
Comment added in [a9bd146](https://github.com/mudita/mudita-center/pull/790/commits/a9bd146d5751575c75fa31d4e2c8fdcd07d0a44d)

Reply:
Will be easier to test  this component with https://www.npmjs.com/package/react-beautiful-dnd-test-utils, if the state mock will be in the parent state 😉 

What do you think to test this component at the end of the sprint 🙏